core/src/main/thrift/tabletserver.thrift

```diff
@@ -164,8 +174,9 @@ service TabletClientService extends client.ClientService {
                                   5:map<string, map<string, string>> ssio,
                                   6:list<binary> authorizations,
                                   7:bool waitForWrites,
-                                  9:i64 batchTimeOut)  throws (1:client.ThriftSecurityException sec),
-  data.MultiScanResult continueMultiScan(2:trace.TInfo tinfo, 1:data.ScanID scanID) throws (1:NoSuchScanIDException nssi),
+                                  9:TSamplerConfiguration samplerConfig,
```

Comment:
This is no good. You need to leave `batchTimeOut` as '9'. Can't change it to 10 w/o breaking old clients.